pkg/lib/files/files.go

```diff
@@ -31,6 +31,57 @@ import (
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
 )
 
+func Open(path string) (*os.File, error) {
+	fileBytes, err := os.Open(path)
+	if err != nil {
+		return nil, errors.Wrap(err, ErrorReadFile(path).Error())
+	}
+
+	return fileBytes, nil
+}
+
+func OpenFile(name string, flag int, perm os.FileMode) (*os.File, error) {
```

Comment:
`flag` doesn't seem to be used. Should we use it, or remove it and hardcode `flag` to the value that's being used now?